src/main/java/com/novare/inventoryManager/menus/cashierMenu/CashierMenuController.java

```diff
@@ -0,0 +1,54 @@
+package com.novare.inventoryManager.menus.cashierMenu;
+
+import com.novare.inventoryManager.employees.EmployeeRole;
+import com.novare.inventoryManager.menus.cashierMenu.CashierMenuModel;
+import com.novare.inventoryManager.menus.cashierMenu.CashierMenuView;
+import com.novare.inventoryManager.utils.Menu;
+
+import java.util.Scanner;
+
+class CashierMenuController {
+    private final CashierMenuModel model;
+    private final CashierMenuView view;
+    private final Scanner scanner;
+
+    CashierMenuController(CashierMenuModel model, CashierMenuView view) {
+        this.model = model;
+        this.view = view;
+        this.scanner = new Scanner(System.in);
+    }
+
+    void requestUserInput() {
+        String input = scanner.nextLine();
+
+        try {
+            int selectedOption = Integer.parseInt(input);
+            handleOption(selectedOption);
+        }
+        catch (NumberFormatException | IndexOutOfBoundsException exception) {
+            Menu.printInvalidOption();
+
+            Menu.printRequest();
+            requestUserInput();
+        }
+    }
+
+    void handleOption(int selectedOption) throws IndexOutOfBoundsException {
+        try {
+            switch (selectedOption) {
+
+                case 1 -> System.out.println("TODO: View the inventory products");
+                case 2 -> System.out.println("TODO: Add orders to the orders list");
+                case 3 -> System.out.println("TODO: Sell an item");
+                case 4 -> System.out.println("TODO: Open the group chat");
+
```

Comment:
Add change password in CashierMenu

Reply:
Move to the tech debt. Will be done later during this sprint or moved to the next